lms/resources/_js_config/__init__.py

```diff
@@ -198,6 +198,25 @@ def enable_lti_launch_mode(self, assignment: Assignment):
         }
         self._config["debug"]["values"] = self._get_lti_launch_debug_values()
 
+        self._config["editing"] = {
+            # Endpoint to get any data needed to get into "editing mode"
+            "getConfig": {
+                "path": self._request.route_path(
+                    "api.assignment.config", assignment_id=assignment.id
+                ),
+                "data": {
+                    "lms": {
+                        "product": self._request.product.family,
+                    },
+                },
+            },
+            # TODO These two probably make more sense in another nested key
+            # Where to submit the eventual re-configure
+            "form_action": self._request.route_url("configure_assignment"),
```

Comment:
This should be `formAction` for consistency with naming of other fields.

It might make sense to change the LMS frontend config to use snake-case everywhere in future to avoid confusion in the backend code from mixing up the two.